_includes/header.html

```diff
@@ -6,7 +6,7 @@
             <li class="menu__entry"><a href="{{ '/about' | prepend: site.baseurl }}">About</a></li>
             <li class="menu__entry"><a href="{{ '/download' | prepend: site.baseurl }}">Download</a></li>
             <li class="menu__entry"><a href="{{ '/tutorial' | prepend: site.baseurl }}">Tutorial</a></li>
-            <li class="menu__entry"><a href="https://github.com/alda-lang/alda/blob/master/doc/index.md">Documentation</a></li>
+            <li class="menu__entry"><a href="/documentation">Documentation</a></li>
```

Comment:
The ideal solution would be to have replace the `documentation.md` to something like `docs.md` which contains all the docs. But, I'm not that experienced with Jekyll so doing that would need a bit more calibration, I couldn't figure out the paths much myself.